extra-scientific/maxima/autobuild/defines

```diff
@@ -4,9 +4,22 @@ PKGDEP="gnuplot texinfo ecl tk rlwrap"
 BUILDDEP="emacs"
 PKGDES="A sophisticated computer algebra system"
 
-AUTOTOOLS_AFTER="--enable-ecl --with-default-lisp=ecl \
-                 --libexecdir=/usr/lib"
+AUTOTOOLS_AFTER="""
+    --prefix=/usr
+    --mandir=/usr/share/man
+    --infodir=/usr/share/info
+    --enable-ecl 
+    --with-default-lisp=ecl 
+    --libexecdir=/usr/lib
+    """
+
 ABSHADOW=no
-MAKE_AFTER="emacsdir=/usr/share/emacs/site-lisp/maxima"
+RECONF=0
 NOPARALLEL=1
 NOLTO=1
+
+ABSPLITDBG=0
```

Comment:
Why?

Reply:
> Why?

It's built by ECL, so no meaningful debugging info.